api/query/cache/service/search.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/web-platform-tests/wpt.fyi/shared"
 )
 
-type searchErr struct {
+type searchError struct {
```

Comment:
Which linter changed this?

Reply:
This was caused by `errname` where error types need to be suffixed with `Error`.